src/main/java/io/jenkins/plugins/orka/RunOnceCloudRetentionStrategy.java

```diff
@@ -82,9 +90,31 @@ public DescriptorImpl getDescriptor() {
         public String getDisplayName() {
             return "Terminate immediately after use";
         }
+
+        public FormValidation doCheckIdleMinutes(@QueryParameter String value) {
```

Comment:
This is the same check you have in `IdleTimeCloudRetentionStrategy`. 
Consider extracting it to a common place and reusing it.

Reply:
Done.